lib/challenge.js

```diff
@@ -34,6 +45,13 @@ var pet_owner;
 
 // Answer Starts Here
 
+// Description: We are making an app for a veterinary clinic - it manages pets' vaccinations.
+var pet_owner = {
+
+  name: "Laurisa",
+  address: "193 Jackson Street"
+  {
```

Comment:
This is an opening brace, not a closing brace. This will lead to a syntax error. However, you don't make this mistake anywhere else, so it's probably just a typo.